creator-node/src/premiumContent/helpers.ts

```diff
@@ -0,0 +1,61 @@
+import type Logger from 'bunyan'
+import { PremiumContentSignatureData, PremiumContentType } from './types'
+
+type PremiumContentMatchArgs = {
+  signedDataFromDiscoveryNode: PremiumContentSignatureData
+  userWallet: string
+  premiumContentId: number
+  premiumContentType: PremiumContentType
+  logger?: Logger
+}
+
+const PREMIUM_CONTENT_SIGNATURE_MAX_TTL = 6 * 60 * 60 * 1000 // 6 hours
+
+export const isPremiumContentMatch = ({
+  signedDataFromDiscoveryNode,
+  userWallet,
+  premiumContentId,
+  premiumContentType,
+  logger
+}: PremiumContentMatchArgs) => {
+  const {
+    premiumContentId: signedPremiumContentId,
+    premiumContentType: signedPremiumContentType,
+    userWallet: signedUserWallet,
+    timestamp: signedTimestamp
+  } = signedDataFromDiscoveryNode
+
+  const theLogger = logger || console
```

Comment:
could just add console as a default value when destructuring instead of doing this